### ISSUE TITLE
Bug: Documentation examples fail to render: /src/index.js: r is not a function (1:0)

### ISSUE THREAD

**Issue by treygilliland** [pixijs/pixijs#9677]
### Current Behavior

When navigating to any of the examples or the playground on the pixi website I receive this error on both `dev` and `v7.3.x`:

<img width="737" alt="Screen Shot 2023-09-20 at 5 32 07 PM" src="https://github.com/pixijs/pixijs/assets/28707707/ad614fa0-5565-4d43-8524-da7302ca28a0">

This error occurs on both Firefox and Chrome.

### Expected Behavior

Examples and playground should render pixijs code.

### Steps to Reproduce

Navigate to the [playground](https://pixijs.com/playground) or any of the [examples](https://pixijs.com/examples) and make sure version `v7.3.x` (default) or `dev` are selected. Canvas will not render but instead show an error with a cryptic message:

```
/src/index.js: r is not a function (1:0)

> 1 | import * as PIXI from 'pixi.js';
      ^
  2 | 
  3 | const app = new PIXI.Application({ background: '#1099bb', resizeTo: window });
  4 | 
  ```

### Environment

- **`pixi.js` version**: v7.3.x
- **Browser & Version**: Version 116.0.5845.96 (Official Build) (arm64) and Firefox Version 117
- **OS & Version**: MacOS
- **Running Example**: https://pixijs.com/playground

### Possible Solution

This is my first time using pixijs so not sure!

**Comment by chenweihuan** [comment]
I had the same problem.

**Comment by barroij** [comment]
Same here, both on Firefox 117.0.1 and Chrome 117.0.5938.92

**Comment by a952543073** [comment]
![image](https://github.com/pixijs/pixijs/assets/22365181/ff19ba3f-59dc-4df6-9054-480dcf277b35)
I'm also in the official demo document and I get an error here

**Comment by Zyie** [comment]
I finally tracked down the issue and should be resolved pretty soon with #9686
